client/components/domains/contact-details-form-fields/index.jsx

```diff
@@ -439,6 +440,17 @@ export class ContactDetailsFormFields extends Component {
 		);
 	}
 
+	renderAlternateEmailFieldForGSuite() {
+		return (
+			<div className="contact-details-form-fields__row g-suite-alternate-mail">
+				<Input
+					label={ this.props.translate( 'Alternate Email' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Alternate Email Address' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).